fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/SchedulerJobsTestResults.java

```diff
@@ -537,11 +537,11 @@ public void testUpdateLoanSummaryJobOutcome() throws InterruptedException {
         final String currentDate = dateFormat.format(todaysDate.getTime());
 
         todaysDate.add(Calendar.MONTH, -1);
-        final String LOAN_DISBURSEMENT_DATE = dateFormat.format(todaysDate.getTime());
+        final String loanDisbursementDateValue = dateFormat.format(todaysDate.getTime());
 
```

Comment:
Is there a reason why we have ...Value added to some (but not all) of these?

Reply:
Updated, I did it since the variable  loanDisbursementDate was used in some files, did not let me mass replace.